### PR TITLE
HomeWork

### PR DESCRIPTION
**Pull request by Daniluko**

### CODE REVIEW

File: Lesson10/src/main/java/Lesson12/MainToJSONMarshaller.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+/**
+ * Created by Rostyslav on 28.01.19
+ */
+public class MainToJSONMarshaller {
+    public static void main(String[] args) {
+        MateGroup mateGroup = MateGroup.mateGroupExampleCreator();
+
+        try {
+            File file = new File("src/main/resources/lesson10/mateGroupMarshaller.xml");
```
Comment (Yanetta) [inline Lesson10/src/main/java/Lesson12/MainToJSONMarshaller.java]: Does it work with xml? I thought tree should be json

Comment (Daniluko) [reply Lesson10/src/main/java/Lesson12/MainToJSONMarshaller.java]: yes, i update this

File: Lesson10/src/test/java/Lesson11/StringChangerServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    @Spy
+    @InjectMocks
+    private StringChangerService stringChangerService;
+
+    @Mock
+    private StringLowerUpperService stringLowerUpperService;
+
+
+    @Before
+    public void setUp() throws Exception {
```
[the remark this answers is not in the document]
Comment (Daniluko) [reply Lesson10/src/test/java/Lesson11/StringChangerServiceTest.java]: has automatically generated

File: Lesson10/src/test/java/Lesson11/StringChangerServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+    public void testAddStartAndEndUpperVerify() {
+        stringChangerService.addStartAndEndUpper("some", "UPPER", "some");
+        verify(stringLowerUpperService).toUpper("UPPER");
+
+    }
+
+    @Test
+    public void addStartAndEndTotalLower() {
+        String expected = "startlover";
```
[the remark this answers is not in the document]
Comment (Daniluko) [reply Lesson10/src/test/java/Lesson11/StringChangerServiceTest.java]: It's my mistake , i have another check with assertEquals.
But accidentally deleted

**Comment by Daniluko** [comment]
> Sorry. Don't cheak - you have conflict in merge

corrected

**Comment by Daniluko** [comment]
> I didn't do changes to my comments. Read there are
Sorry, corrected

**Pull request merged**